### PR TITLE
.gitignore: ignore jellyfin-web symlink & .vscode: dotnet telemetry optout

### PR DESCRIPTION
**Pull request by telans**
**Changes**
Captures a symlink for MediaBrowser.WebDashboard/jellyfin-web & removes duplicate entry.
Adds dotnet telemetry optout environment variable for vscode tasks

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
-   // Use IntelliSense to find out which attributes exist for C# debugging
-   // Use hover for the description of the existing attributes
-   // For further information visit https://github.com/OmniSharp/omnisharp-vscode/blob/master/debugger-launchjson.md
-   "version": "0.2.0",
-   "configurations": [
+    // Use IntelliSense to find out which attributes exist for C# debugging
+    // Use hover for the description of the existing attributes
+    // For further information visit https://github.com/OmniSharp/omnisharp-vscode/blob/master/debugger-launchjson.md
```
Comment (dkanada) [inline .vscode/launch.json:4]: Can we delete these comments so GitHub stops complaining?

**Comment by dkanada** [comment]
Are the Windows tests failing for a specific reason in this pull request?

**Comment by telans** [comment]
```
[08:27:41 ERR] An exception occurred while iterating over the results of a query for context type 'Jellyfin.Server.Implementations.JellyfinDb'.
System.ObjectDisposedException: Cannot access a disposed object.
Object name: 'IServiceProvider'.
```

Unclear how removing comments introduces that

**Comment by telans** [comment]
All I did was rebase 🤷

**Pull request merged**